src/components/CompareResults/ResultsTable.tsx

```diff
@@ -16,17 +16,25 @@ import ResultsControls from './ResultsControls';
 import TableContent from './TableContent';
 import TableHeader from './TableHeader';
 
-const cellsConfiguration: CompareResultsTableConfig[] = [
+const cellsConfiguration: CompareResultsTableConfig = [
   {
     name: 'Platform',
     disable: true,
     filter: true,
     key: 'platform',
     gridWidth: '2fr',
-    possibleValues: ['Windows', 'OSX', 'Linux', 'Android'],
-    matchesFunction: (result: CompareResultsItem, value: string) => {
+    possibleValues: [
+      { label: 'Windows', key: 'windows' },
+      { label: 'OSX', key: 'osx' },
+      { label: 'Linux', key: 'linux' },
+      { label: 'Android', key: 'android' },
+    ],
+    matchesFunction(result, valueKey) {
+      const label = this.possibleValues.find(
+        ({ key }) => key === valueKey,
+      )?.label;
```

Comment:
Initially I wasn't a big fan of these `find` operations, because this function is called in a loop on each of the results.
But then I profiled and found this to be negligible compared to the full react rendering, so I decided that we may optimize this later...